pkg/console/operator/operator.go

```diff
@@ -175,10 +187,19 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 		return err
 	}
 
+	proxyConfig, err := c.proxyCfgLister.Get("cluster")
+	if err != nil {
+		if !errors.IsNotFound(err) {
```

Comment:
👍 The proxy config might not be there always on upgrade, and we shouldn't error. Good catch.

I believe we can remove this in 4.3.